frontend/packages/topology/src/types.ts

```diff
@@ -225,12 +225,16 @@ export interface Controller extends WithState {
 }
 
 type ElementEvent = { target: GraphElement };
+export type ElementEventListener = EventListener<[ElementEvent & { child: GraphElement }]>;
+
 export type ElementChildEventListener = EventListener<[ElementEvent & { child: GraphElement }]>;
 
 export type NodeCollapseChangeEventListener = EventListener<[{ node: Node }]>;
 
 export type GraphLayoutEndEventListener = EventListener<[{ graph: Graph }]>;
 
+export const HIDE_ELEMENT_EVENT = 'element-hide';
+export const SHOW_ELEMENT_EVENT = 'element-show';
```

Comment:
Would prefer a single event and perhaps a payload to identify the new visibility state as a boolean value.